core/metacling/src/CMakeLists.txt

```diff
@@ -38,8 +38,23 @@ ROOT_LINKER_LIBRARY(Cling
         $<TARGET_OBJECTS:MetaCling>
         LIBRARIES ${CLING_LIBRARIES})
 
+if(MSVC)
+  set_target_properties(Cling PROPERTIES WINDOWS_EXPORT_ALL_SYMBOLS TRUE)
+  set(cling_exports ??1Value@cling@@QAE@XZ
+      ?setValueNoAlloc@internal@runtime@cling@@YAXPAX00D_K@Z
+      ?printValue@cling@@YA?AV?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@std@@PB_N@Z
+      ?EvaluateDynamicExpression@internal@runtime@cling@@YA?AVValue@3@PAVInterpreter@3@PAVDynamicExprInfo@123@PAVDeclContext@clang@@@Z
+  )
```

Comment:
This looks quite hacky, is it the only way?

Reply:
Well, there are always several solutions for a given problem, but I would say this is the simplest one